native-schema-registry/csharp/AWSGsrSerDe/AWSGsrSerDe.Tests/serializer/GlueSchemaRegistryKafkaSerializerTests.cs

```diff
@@ -96,14 +107,27 @@ public void KafkaSerDeTestForAllProtobufTypes(IMessage message)
                 { GlueSchemaRegistryConstants.DataFormatType, GlueSchemaRegistryConstants.DataFormat.PROTOBUF },
             };
 
-            var glueSchemaRegistryKafkaSerializer = new GlueSchemaRegistryKafkaSerializer(configs);
-            var glueSchemaRegistryKafkaDeserializer = new GlueSchemaRegistryKafkaDeserializer(configs);
+            KafkaSerializer.Configure(configs);
+            KafkaDeserializer.Configure(configs);
 
-            var serialized = glueSchemaRegistryKafkaSerializer.Serialize(message, message.Descriptor.FullName);
+            var serialized = KafkaSerializer.Serialize(message, message.Descriptor.FullName);
 
             var deserializedObject =
-                glueSchemaRegistryKafkaDeserializer.Deserialize(message.Descriptor.FullName, serialized);
+                KafkaDeserializer.Deserialize(message.Descriptor.FullName, serialized);
             Assert.AreEqual(message, deserializedObject);
         }
+
+
+        // TODO: Improve and automate the memory leak check
+        // Test for detecting memory leak
+        // [Test]
+        // [TestCaseSource(nameof(TestMessageProvider))]
+        // public void KafkaSerDeTestForAllProtobufTypesMemCheck(IMessage message)
```

Comment:
I suggest creating a separate test file for these. You can refactor this in next PR.

Reply:
sounds good